TimeTrackerApp/TimeTrackerApp/Utilites/Extensions/Font+Extension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Font+Extension.swift
+//  TimeTrackerApp
+//
+//  Created by Sevda Gul Baran on 13.09.2022.
+//
+
+import Foundation
+import UIKit
+
+
+extension UIFont{
+    
+    static func appFont(size: CGFloat, type:String) -> UIFont
+    {
+        return UIFont(name: type, size: size)!
+    }
```

Comment:
parantezi fonksiyonun yanında başlatıyoruz, bu şirketten şirkete değişebilir tabii :D 
```suggestion
    static func appFont(size: CGFloat, type: String) -> UIFont {
        UIFont(name: type, size: size)!
    }
```

Reply:
swiftliyoruzda böyle :P